### ISSUE TITLE
Terrain graphics feature gating

### ISSUE THREAD

**Issue by cosmic-linden** [secondlife/viewer#711]
### Test plan
- needed.

@RunitaiLinden 's proposal:

Potato mode - base color only
Low - 16-texture sample (i.e. no emissive)
Mid - All textures, no triplanar
High - Triplanar

Profile the different graphics levels to make sure there is actually a performance benefit to reducing material fidelity

GLTF specification specifies the order of dropped ORM features:

https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#additional-textures

**Comment by cosmic-linden** [comment]
Profiling looks promising on my machine and shows that dropping textures is an effective way to improve performance of PBR terrain, with one exception: dropping just the occlusion channel is not much of a performance win, because the occlusion channel is baked into the metallic-roughness texture. When dropping all textures except base color, performance is comparable to non-PBR.

```
0.70ms : PBR triplanar + full ORM+emissive
0.53ms : -triplanar
0.43ms : -emissive
0.42ms : -occlusion
0.27ms : -normals
0.15ms : -metallic/roughness
0.17ms: non-PBR
```

Order of dropped features is in accordance with the GLTF spec when possible. Dropping metallic/roughness is off-spec.

To summarize, here are the costs of terrain (reasonable near-upper-bounds) for the proposed settings:

- Potato mode - 0.15ms - base color only
- Low - 0.43ms - 16-texture sample (i.e. no emissive)
- Mid - 0.53ms - All textures, no triplanar
- High - 0.70ms - Triplanar

**Comment by georgebproductengine** [comment]
Passed QA.

Verified on the Second Life Release 7.1.8.9103842320 (64bit) on Win10/OSX in the scope of https://github.com/secondlife/iqa/issues/239.
